### PR TITLE
Add LM tuning functionality & detailed transcription output

### PR DESCRIPTION
**Pull request by ryanleary**
`transcribe.py` is modified to take a `--word_json` parameter which returns a json-formatted transcription that includes the top path string, as well as all requested paths tokenized with word start/stop times and confidences (currently broken).

`test.py` is modified to optionally be able to skip decoding and simply write the logits to disk.

`tune_decoder.py` reads the written logits in and in parallel sweeps the alpha and beta parameters so that an optimal value may be chosen.

### CODE REVIEW

**Comment by SeanNaren** [comment]
Thanks man, I think once we get the LM stuff sussed out a README update is definitely a priority!
